infrastructure/cdn-in-a-box/Makefile

```diff
@@ -180,7 +180,7 @@ $(TS_DIST_RPM): $(TS_SOURCE)
 	"$(PKG_COMMAND)" $(PKG_FLAGS) traffic_stats$(BUILD_SUFFIX)
 
 $(ORT_DIST_RPM): $(ORT_SOURCE)
-	"$(PKG_COMMAND)" $(PKG_FLAGS) cache-config$(BUILD_SUFFIX)
+	"$(PKG_COMMAND)" -b $(PKG_FLAGS) cache-config$(BUILD_SUFFIX)
```

Comment:
No need to manually build [`apache/trafficcontrol-cache-config_builder`](https://hub.docker.com/r/apache/trafficcontrol-cache-config_builder), the image exists in Docker Hub.

Reply:
I made those changes to make the Github checks pass. The build was failing because the Docker Hub images don't have the Dockerfile changes in this PR: https://github.com/apache/trafficcontrol/pull/5878/files#diff-bbc5344acb7207878d9fba88a12ae8a6149729c41b66a7e62377453133f1a208R46

Is there a procedure for getting Github checks to pass when a Dockerfile needs updating?

Reply:
Gotcha, I see you added `pandoc` as a dependency. I have manually pushed a build for the `apache/trafficcontrol-cache-config_builder` image from this branch, so our CI should pass now.

Reply:
> Is there a procedure for getting Github checks to pass when a Dockerfile needs updating?

Not an automated one ATM, although we would definitely benefit from having one in the future.

Reply:
Thanks! Ok, I removed the commits changing it to build instead of pulling